common/math/geometry/include/geometry/solver/polynomial_solver.hpp

```diff
@@ -84,6 +84,11 @@ class PolynomialSolver
  */
   double linearFunction(double a, double b, double t) const;
 
+  /**
+   * @brief Hard coded parameter, tolerance of calculation results of the PolynomialSolver
+   */
+  constexpr static double tolerance = 1e-7;
```

Comment:
You should describe why this value was determined.
For example, "This value was determined by @hakuturu583 based on personal experience and not on any specific technical basis."
In particular, based on this change, you need to state why it is inappropriate in `std::numeric_limits<double>::epsilon`.

Reply:
I described.

- Who decided this hard-coded parameter.
- Why the hard coded parameter should not std::numeric_limits<double>::epsilon.
- Why the hard coded parameter value is 1e-7.

On the comment.